inc/class_parser.php

```diff
@@ -1709,18 +1709,27 @@ function text_parse_message($message, $options=array())
 
 		$find = array(
 			"#\[(b|u|i|s|url|email|color|img)\](.*?)\[/\\1\]#is",
+			"#\[(email|color|size|font|align|video|)=[^]]*\](.*?)\[/\\1\]#is",
```

Comment:
`(email|color|size|font|align|video|)` there's an extra `|` at the end of this regex which must be removed.

Reply:
Yes, you are right, I missed that, I will fix that ASAP, thank you for reviewing ;)